go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/VictoriaMetrics/fastcache v1.10.0
-	github.com/ava-labs/avalanchego v1.10.1
+	github.com/ava-labs/avalanchego v1.10.2
```

Comment:
Can we bump to the latest v1.10.3?

Reply:
I specifically downgraded to 1.10.2 as avalanchego 1.10.3 requires go-ethereum 1.11.4 which we haven't synced with this repo yet
https://github.com/ava-labs/subnet-evm/pull/681/commits/a7d8b20973f915249fe210d11f346fad3cd52d26